docs/source/reservations/existing-machines.rst

```diff
@@ -151,3 +152,53 @@ To clean up all state created by SkyPilot on your machines, use the ``--cleanup`
     sky local up --ip $IP_FILE --ssh-user SSH_USER --ssh-key-path $SSH_KEY --cleanup
 
 This will stop all Kubernetes services on the remote machines.
+
+
+Setting up Multiple Clusters
+----------------------------
+
+You can set up multiple Kubernetes clusters with SkyPilot by using different ``context-name`` values for each cluster and merging the kubeconfigs:
+
+.. code-block:: bash
+
+    # Set up first cluster and save the kubeconfig
+    sky local up --ips cluster1-ips.txt --ssh-user user1 --ssh-key-path key1.pem --context-name cluster1
+    cp ~/.kube/config ~/.kube/config.cluster1
+
+    # Set up second cluster
+    sky local up --ips cluster2-ips.txt --ssh-user user2 --ssh-key-path key2.pem --context-name cluster2
+    cp ~/.kube/config ~/.kube/config.cluster2
+
+    # Combine the kubeconfigs into ~/.kube/config
+    KUBECONFIG=~/.kube/config.cluster1:~/.kube/config.cluster2 kubectl config view --flatten > ~/.kube/config
```

Comment:
Why don't our `local up` just add new context in the existing `kubeconfig`? Just like what `gcloud containers clusters get-credentials` would do?

Reply:
that's a great idea, just required some handling. I've changed our code to update contexts directly now.